client/src/app.jsx

```diff
@@ -1,9 +1,35 @@
-import React from 'react';
+  import React, { Component } from 'react';
+  import axios from 'axios';
 
-const App = () => (
-  <div>
-    App me down!
-  </div>
-);
+  export default class App extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        listingId: null
+      };
+      this.houseData = {};
```

Comment:
I would move house data into props as no information should be kept in a react component outside of state.